### ISSUE TITLE
Keyboard navigation with arrow keys does not work when grid has group rows.

### ISSUE THREAD

**Issue by MayaKirova** [IgniteUI/igniteui-angular#2310]
### Description  
When navigating via arrow keys up/down in a grid that has group rows, navigation breaks after moving through a group row.

### Steps to reproduce  

1. Open the Grid GroupBy sample from demos dev.
2. Group by a column.
3. Select a data cell and start navigating down/up via the arrow keys.

### Result  
Navigation breaks after moving through a group row. Cell loses focus and grid starts scrolling.

### Expected result  
Navigation should continue and the next cell should be selected and focused..
